v2/internal/testcommon/test_context.go

```diff
@@ -177,6 +177,7 @@ func createTestNamespaceName(t *testing.T) string {
 
 	name := strings.ReplaceAll(strings.ToLower(t.Name()), "/", "-")
 	result := "aso-" + strings.ReplaceAll(name, "_", "-")
+	result = strings.ReplaceAll(result, ".", "-")
```

Comment:
could use regexp(`[./_]`) here to match and replace all at once?

Reply:
Done